### ISSUE TITLE
What is the latest version of YOLO? Is V5 a scam?

### ISSUE THREAD

**Issue by franva** [pjreddie/darknet#2198]
Hi guys,

I am learning the YOLO, it looks great~!
I think this Github repo and [this website](https://pjreddie.com/darknet/yolo/) are the official websites for YOLO?

But recently I saw [an article ](https://towardsdatascience.com/how-to-train-a-custom-object-detection-model-with-yolo-v5-917e9ce13208)talking about YOLO V5.

My 1st thought is : what? Since what time does YOLO get V5??

I searched from the Cornell University and could not find anything about YOLO V5 either.

1. Could someone tell me what is the latest version of YOLO?
2. What is the official website for YOLO?

Appreciated,

Winston

**Comment by AlexeyAB** [comment]
**The latest version - YOLOv4**, with paper, with URLs from official repository, and with the best Accuracy/Speed among all known algorithms.

YOLOv5-Ultralytics - is just a name, model is worse than YOLOv4, without improvements, without a scientific article, and with fake comparisons in a couple of blogs.

* Negative opinion of Deep Learning developers about Yolov5-Ultralytics: https://www.reddit.com/r/MachineLearning/comments/h0ddia/news_yolov5_is_here_stateoftheart_object/

* Official site (v3 is Joseph’s latest version of Yolo, after which Joseph stopped developing Yolo): https://pjreddie.com/darknet/yolo/

* Xnor AI (the company develops Yolo) was bought by Apple for $200 M: https://www.forbes.com/sites/janakirammsv/2020/01/19/apple-acquires-xnorai-to-bolster-ai-at-the-edge/#20a12e943975

* Official repository - declares **the latest version of YOLOv 4**: https://github.com/pjreddie/darknet#darknet

* Official **YOLOv4 repository**: https://github.com/AlexeyAB/darknet

* Official **paper - YOLOv4**: https://arxiv.org/abs/2004.10934


----

* The official statement that the new Yolo repository is https://github.com/AlexeyAB/darknet https://twitter.com/pjreddie/status/1253891078182199296

![image](https://user-images.githubusercontent.com/4096485/84315695-b9337480-ab72-11ea-95dc-afcf6a98d2ae.png)

----

* **YOLOv5 is worse than YOLOv4**: 

![84604438-abf6ec80-aec8-11ea-8341-f4563ea51dbc](https://user-images.githubusercontent.com/4096485/85428346-ba5a9d80-b585-11ea-84b7-24d87df6dc90.png)


----

* Comparison of YOLOv4 with other models: 

![comparison_gpus](https://user-images.githubusercontent.com/4096485/85428417-d9592f80-b585-11ea-9e65-f8e950818122.png)


----

* Speed (FPS) of YOLOv4 in different libraries with different batch-size - up to 400 FPS:

![84310129-d57ee380-ab69-11ea-87c6-e7f2c33d5f40](https://user-images.githubusercontent.com/4096485/85430151-6ac9a100-b588-11ea-803f-4699827080b7.png)

**Comment by franva** [comment]
Firstly, thank you for your detailed explanation about where YOLO comes from, who firstly invented YOLO and how YOLO is currently going.
This really clears my mind about the history of YOLO and who is the official and who is not, so thumb up for this :)

After knowing these information, as a beginner of AI I can tell why the article I referenced in my question is so confusing.

In the [video](https://www.youtube.com/watch?reload=9&v=MdF6x6ZmLAY&t=520) mentioned in the article, they claimed that their fake YOLOV5 aiming at different perspectives ,e.g. ease of use, exportability, memory requirement, etc...
![image](https://user-images.githubusercontent.com/5151146/85479403-6e4d3e80-b601-11ea-806e-f1977870f4c9.png)

With not much AI background knowledge, as a beginner, I don't think their fake YOLOV5 should use the name YOLO at all.
If a new model architecture is created, then I would expect to see its related academic papers on some scientific websites.
If it's just a tweak of an existing YOLO V3, the author of fake YOLO V5 should name it something like YOLOV3-PyTorch.

As a beginner of AI, I dislike this kind of behavior, as it is waste of my time to learn such a fake architecture and I am sure there are many beginners who will have similar experience as mine, but not all of them have luck to discovered that "YOLOV5" is a fake one. So as an owner of YOLO V4, and the owner of YOLO V3 and all other owners of YOLO should do something to make people aware of this wrongdoing.

So here is my thought, I feel that many of users(developers) will be familiar with Python and PyTorch framework based AI models versus C based YOLO models.  This is where the selling point of the fake YOLO V5 is. Therefore, why not the owner(@AlexeyAB ) of this YOLO V4, create an authentic YOLO V4 for PyTorch(Python) version model? I feel this is a better way to defeat the fake one.

**Comment by AlexeyAB** [comment]
I am not opposed the Ultralytics repository. I am opposed to unfair comparisons with YOLOv4.
YOLOv4 can be trained on both repos: https://github.com/ultralytics/yolov3 and https://github.com/ultralytics/yolov5
So in the future, a more accurate and faster version of YOLO on these repositories may indeed be released. And it will be very convenient for those who want to use Python and Pytorch.

**Comment by AlexeyAB** [comment]
YOLOv4 training and inference on different frameworks / libraries:

**Pytorch**-implementations:
* https://github.com/WongKinYiu/PyTorch_YOLOv4
* https://github.com/Tianxiaomo/pytorch-YOLOv4
* 3D-rotated-bboxes: https://github.com/maudzung/Complex-YOLOv4-Pytorch

**TensorFlow**: https://github.com/hunglc007/tensorflow-yolov4-tflite

**OpenCV** (YOLOv4 built-in OpenCV): https://github.com/opencv/opencv

**TensorRT:** https://github.com/ceccocats/tkDNN

**Tencent/NCNN:** https://github.com/Tencent/ncnn

**TVM** https://tvm.ai/about

**OpenDataCam:** https://github.com/opendatacam/opendatacam#-hardware-pre-requisite

**BMW-InnovationLab** - Training with YOLOv4 has never been so easy (monitor it in many different ways like TensorBoard or a custom REST API and GUI): 
* Training: https://github.com/BMW-InnovationLab/BMW-YOLOv4-Training-Automation
* Inference on GPU: https://github.com/BMW-InnovationLab/BMW-YOLOv4-Inference-API-GPU
* Inference on CPU: https://github.com/BMW-InnovationLab/BMW-YOLOv4-Inference-API-CPU
